modules/administration_manual/pages/appliance/Clamav.adoc

```diff
@@ -9,31 +9,29 @@ Appliance. It is composed of two parts:
 [[install-clamav-and-related-components]]
 == Install ClamAV and Related Components
 
-First, start the appliance and go to ``**System and domain settings**''.
+First, start the appliance and go to ``**System and domain settings**``.
```

Comment:
The text shouldn' be wrapped in double backticks, but with double-quotes, as the user's being directed to click on a specific section of the UI. Secondly, AsciiDoc only requires single backticks for inline code.